exporter/opentelemetry-exporter-otlp-proto-http/tests/test_proto_log_exporter.py

```diff
@@ -167,6 +171,82 @@ def test_exporter_env(self):
         )
         self.assertIsInstance(exporter._session, requests.Session)
 
+    @patch("requests.Session.post")
+    def test_exported_log_without_trace_id(self, mock_post):
```

Comment:
These two tests are also very similar. What do you think about refactoring them too?